oeps/oep-0001.rst

```diff
@@ -0,0 +1,335 @@
+=================================
+OEP-1: OEP Purpose and Guidelines
+=================================
+
++--------+---------------------------------------+
+|OEP     | :doc:`oep-1`                          |
++--------+---------------------------------------+
+|Title   | OEP Purpose and Guidelines            |
++--------+---------------------------------------+
+|Author  | Calen Pennington <cale@edx.org>       |
++--------+---------------------------------------+
+|Status  | Approved                              |
++--------+---------------------------------------+
+|Type    | Process                               |
++--------+---------------------------------------+
+|Created | 26-Mar-2016                           |
++--------+---------------------------------------+
+|Arbiter | Eddie Fagin <efagin@edx.org>          |
++--------+---------------------------------------+
+
+
+What is an OEP?
+===============
+
+OEP (pronounced "oh-epp") stands for Open edX (Enhancement) Proposal. An OEP is a
+document that specifies a specific choice being made by the
+Open edX community, in the form of a best-practice, architecture
+design, technology decision, or process adjustment. It should
+provide the use-cases and rationales that surround that choice.
+OEPs are not the only way for a change to be made to Open edX,
+however the goal is for it to be a repository collecting the
```

Comment:
what is "it" in this case? this sentence seems to imply a broader notion of a knowledge archive for all of the individual proposals